### DISCUSSION TITLE
Installing SO in RHEL7 on airgap network

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#7874] [Unsupported Versions] [answered]
Is this possible? I'm planning on testing mounting SO Centos image on RHEL7 and run"./so-setup iso" and see where it takes me.  Am I on the right direction? I just want the simple "Standalone" all-in-one configuration.

**Comment by dougburks** [comment, marked as answer]
RHEL is currently untested and unsupported.
